edivorce/apps/core/templates/dashboard/print_form.html

```diff
@@ -27,13 +27,13 @@ <h1>Review and Print your Divorce Forms</h1>
              aria-expanded="false" data-target="#collapseF1"
              aria-controls="collapseF1">
             <div>
-                <span>Notice of Joint Family Claim Form (F1)</span>
+                <span>Notice of Joint Family Claim Form (F1) and Party's Certificate</span>
             </div>
         </div>
         <p class="review-buttons">
             <a href="{% url 'pdf_form' '1' %}" class="btn btn-primary spinner"
-               target="_blank"><i class="fa fa-print" aria-hidden="true"></i>
-                Review and Print
+               target="_blank"><i class="fa fa-pencil" aria-hidden="true"></i>
+                Review, Print & Sign
```

Comment:
Should be `Review, Print &amp; Sign`

Reply:
I guess it actually doesn't make a difference since it appears 5 other times in the file without escaping the entity.  I'll mark it as approved,

Reply:
Good point, I'll fix them all up in the next PR